.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,16 @@
+name: RuboCop
+on: [pull_request]
+env:
+  BUNDLE_WITHOUT: assets:console:development:dynflow_sidekiq:ec2:gce:journald:jsonp:libvirt:openid:openstack:ovirt:redis:service:telemetry:test:vmware
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+          bundler-cache: true
+      - name: Run rubocop
+        run: bundle exec rake rubocop
```

Comment:
I would consider the [Github Actions formatter](https://docs.rubocop.org/rubocop/1.4/formatters.html#github-actions-formatter). Haven't tried it myself and perhaps you need a newer Rubocop version than we use now.

Reply:
Yeah, that is definitelly out of reach now, but I'm all in for upgrading the rubocop once we get rid of hound :+1: